### DISCUSSION TITLE
Supporting businesses that do not operate as a sign of protest

### DISCUSSION THREAD

**Discussion by kiumad** [WeAreMahsaAmini/Core#108] [Ideas]
A list of online shops and local businesses that do not operate these days as a sign of protest and have no income.

People can support them later

But there is also an issue, this is a complete list that the government may use later and harass them

What do you think?

**Comment by AnonymousMaximus2** [comment]
> A list of online shops and local businesses that do not operate these days as a sign of protest and have no income.
> 
> People can support them later
> 
> But there is also an issue, this is a complete list that the government may use later and harass them
> 
> What do you think?

Create an economy independent from government economy, that government can not fine, or sanction or block their assets, ....

**Comment by kaveh6** [comment]
I'm thinking of a reverse idea , to share businesses and shops that did not close their businesses and didnt join to the strikes .

**Reply by ghost** [reply]
They may join later, it's not a good idea to make them enemies. This leads to less unity.

**Comment by whitewetwolf** [comment]
I think we need to provide financial help to individuals/businesses that decide to support the general strike. This involves 

- collecting money (in Europe/US/Canada/etc)
- transferring to Iran
- distributing it between those eligible

I think the third steps is the hardest. How can we find the people who are really participating? How can we protect those in Iran who are helping with the distribution. We can't use any central organization. Moving larges sums money is very easy for the government to track. We need to build the infrastructure so the money can be transferred in a distributed fashion.
